cmd/bucket-replication.go

```diff
@@ -724,7 +724,7 @@ const (
 // matches k1 with all keys, returns 'true' if one of them matches
 func equals(k1 string, keys ...string) bool {
 	for _, k2 := range keys {
-		if strings.EqualFold(strings.ToLower(k1), strings.ToLower(k2)) {
+		if strings.EqualFold(k1, k2) {
```

Comment:
isn't it faster to just do 

```
if strings.ToLower(k1) == strings.ToLower(k2) {
}
```

Reply:
@harshavardhana No, `ToLower` allocates.

Reply:
(also EqualFold can abort as soon as a mismatch is found, not needing to lowercase the rest)

Reply:
okay

Reply:
✅